diff --git a/node_modules/react-redux-toastr/lib/ReduxToastr.js b/node_modules/react-redux-toastr/lib/ReduxToastr.js
index e711aab..2ff4937 100644
--- a/node_modules/react-redux-toastr/lib/ReduxToastr.js
+++ b/node_modules/react-redux-toastr/lib/ReduxToastr.js
@@ -126,7 +126,24 @@ var ReduxToastr = /*#__PURE__*/function (_React$Component) {
     value: function _renderToastrForPosition(position) {
       var _this2 = this;
 
-      var toastrs = this.props.toastr.toastrs;
+      var toastrsNonUnique = Object.assign([], this.props.toastr.toastrs);
+      var toastrs = [];
+
+      if (this.props.preventDuplicates) {
+        toastrsNonUnique = toastrsNonUnique.reverse();
+        for (var i = 0; i < toastrsNonUnique.length; i += 1) {
+          if (!toastrsNonUnique[i].processed) {
+            toastrs.push(toastrsNonUnique[i]);
+          }
+          for (var j = i + 1; j < toastrsNonUnique.length; j += 1) {
+            if (toastrsNonUnique[i].type === toastrsNonUnique[j].type && toastrsNonUnique[i].title === toastrsNonUnique[j].title && toastrsNonUnique[i].message === toastrsNonUnique[j].message) {
+              toastrsNonUnique[j].processed = true;
+            }
+          }
+        }
+      } else {
+        toastrs = toastrsNonUnique;
+      }
 
       if (toastrs) {
         return toastrs.filter(function (item) {
